x/emissions/keeper/keeper_test.go

```diff
@@ -52,6 +52,13 @@ func (s *KeeperTestSuite) SetupTest() {
 	s.msgServer = keeper.NewMsgServerImpl(s.emissionsKeeper)
 	s.mockCtrl = ctrl
 	s.key = key
+
+	// Add all tests addresses in whitelists
```

Comment:
when writing the unit tests, add here a newly generated addr that is not a whitelist admin for testing failing cases